main.js

```diff
@@ -0,0 +1,21 @@
+ var mySwiper = new Swiper ('.swiper-container', {
+    // Optional parameters
+    direction: 'horizontal',
+    loop: true,
+
+    // If we need pagination
+    pagination: {
+      el: '.swiper-pagination',
+    },
+
+    // Navigation arrows
+    navigation: {
+      nextEl: '.swiper-button-next',
+      prevEl: '.swiper-button-prev',
+    },
+
+    // And if we need scrollbar
+    scrollbar: {
```

Comment:
You can remove the `scrollbar: { ... }` from here since you are not using it and also all the comments, you pasted them in from the example but you don't need them